tools/msi/product.wxs

```diff
@@ -1,28 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
-  <?define ProductName="NexPloit CLI" ?>
+  <?define ProductName="Bright CLI" ?>
   <?define ProductAuthor="Bright Security Inc." ?>
-  <?define ProductDescription="$(var.ProductName) is a CLI tool that can initialize, stop, polling and maintain scans in $(var.ProductAuthor)s solutions (Such as NexPloit)." ?>
+  <?define ProductDescription="$(var.ProductName) is a CLI tool that can initialize, stop, polling and maintain scans in $(var.ProductAuthor)s solutions (Such as Bright)." ?>
   <?define RegistryKeyPath="SOFTWARE\$(var.ProductName)" ?>
   <Product Id="*" Name="$(var.ProductName)" UpgradeCode="74b15b74-a063-4524-ae83-69ad421be456" Language="1033" Codepage="1252" Version="$(var.ProductVersion)" Manufacturer="$(var.ProductAuthor)">
     <Package InstallerVersion="200" Compressed="yes" InstallScope="perMachine" />
     <Property Id="PREVIOUSVERSIONSINSTALLED" Secure="yes" />
-    <Property Id="ARPPRODUCTICON" Value="NexPloitIcon" />
-    <Property Id="ApplicationFolderName" Value="nexploit-cli" />
+    <Property Id="ARPPRODUCTICON" Value="BrightIcon" />
+    <Property Id="ApplicationFolderName" Value="bright-cli" />
     <Property Id="WIXUI_INSTALLDIR" Value="INSTALLDIR" />
     <Property Id="INSTALLDIR" Secure="yes">
       <RegistrySearch Id="InstallPathRegistry" Type="raw" Root="HKLM" Key="$(var.RegistryKeyPath)" Name="InstallPath" />
       <RegistrySearch Id="InstallPathRegistryCU" Type="raw" Root="HKCU" Key="$(var.RegistryKeyPath)" Name="InstallPath" />
     </Property>
-    <Media Id="1" Cabinet="nexploitcli.cab" EmbedCab="yes" />
-    <CustomAction Id="NewVersionError" Error="A later version of $(var.ProductName) is already installed. Setup will now exit." />
-    <Upgrade Id="74b15b74-a063-4524-ae83-69ad421be456">
-      <UpgradeVersion Property="PREVIOUSVERSIONSINSTALLED" IncludeMinimum="no" IncludeMaximum="yes" Minimum="0.0.0" Maximum="$(var.ProductVersion)" />
-    </Upgrade>
+    <Media Id="1" Cabinet="brightcli.cab" EmbedCab="yes" />
+    <MajorUpgrade DowngradeErrorMessage="A later version of $(var.ProductName) is already installed. Setup will now exit." />
     <InstallExecuteSequence>

```

Comment:
You can avoid using the `InstallExecuteSequence` element. Instead, use the `Schedule` attribute setting it to `afterInstallExecute`.